src/Profiles.ts

```diff
@@ -409,30 +404,28 @@ export class Profiles {
         if (chosenProfile === "") {
             let newprofile: any;
             let profileName: string;
-            if (quickpick.value) {
-                profileName = quickpick.value;
-            }
+            if (quickpick.value) { profileName = quickpick.value; }
 
             const options = {
-                placeHolder: localize("createNewConnection.option.prompt.profileName.placeholder", "Connection Name"),
-                prompt: localize("createNewConnection.option.prompt.profileName", "Enter a name for the connection"),
+                placeHolder: localize("createZoweSession.option.prompt.profileName.placeholder", "Connection Name"),
+                prompt: localize("createZoweSession.option.prompt.profileName", "Enter a name for the connection"),
                 value: profileName
             };
             profileName = await vscode.window.showInputBox(options);
             if (!profileName) {
-                vscode.window.showInformationMessage(localize("createNewConnection.enterprofileName",
+                vscode.window.showInformationMessage(localize("createZoweSession.enterprofileName",
                     "Profile Name was not supplied. Operation Cancelled"));
                 return;
             }
             chosenProfile = profileName.trim();
             globals.LOG.debug(localize("addSession.log.debug.createNewProfile", "User created a new profile"));
-            try {
-                newprofile = await Profiles.getInstance().createNewConnection(chosenProfile);
-            } catch (error) { await errorHandling(error, chosenProfile, error.message); }
+            const defaultProfile = DefaultProfileManager.getInstance().getDefaultProfile("zosmf");
```

Comment:
hard coded "zosmf" here too

Reply:
Just removed this as when creating a new profile it shouldn't get default values from an existing profile.